groovy/gradle.htmlui/src/org/netbeans/modules/gradle/htmlui/HtmlJavaApplicationProjectWizard.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.netbeans.modules.gradle.htmlui;
+
+import org.netbeans.modules.gradle.spi.newproject.SimpleGradleWizardIterator;
+import org.netbeans.modules.gradle.spi.newproject.TemplateOperation;
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.netbeans.api.project.Project;
+import org.netbeans.api.templates.TemplateRegistration;
+import org.netbeans.modules.gradle.api.NbGradleProject;
+import org.openide.WizardDescriptor;
+import org.openide.filesystems.FileObject;
+import org.openide.filesystems.FileUtil;
+import org.openide.util.Exceptions;
+import org.openide.util.NbBundle.Messages;
+
+@TemplateRegistration(folder="Project/Gradle", position=999, displayName="#template.htmlUIProject", iconBase="org/netbeans/modules/gradle/htmlui/DukeHTML.png", description="description.html")
+@Messages("template.htmlUIProject=Java Frontend Application")
+public class HtmlJavaApplicationProjectWizard extends SimpleGradleWizardIterator {
+    private static final String DEFAULT_LICENSE_TEMPLATE = "/Templates/Licenses/license-default.txt"; //NOI18N
+    
+    @Messages("LBL_FrontendApplicationProject=Java Frontend Application with Gradle")
+    public HtmlJavaApplicationProjectWizard() {
+        super(Bundle.LBL_FrontendApplicationProject(), initParams());
+    }
+
+    private static Map<String, Object> initParams() {
```

Comment:
I guess this method is a copy paste and does not really needed.

Reply:
The method is used on line 47, but the properties are likely useless: ee258ebd978a